source/toolkits/S3Tk.cpp

```diff
@@ -7,6 +7,7 @@
 #include "toolkits/StringTk.h"
 #include "toolkits/UnitTk.h"
 #include "toolkits/TerminalTk.h"
+#include "toolkits/S3CredentialStore.h"
```

Comment:
alphabetic order.
(shame on me, because i'm just noticing here that the includes above this are also not in alphabetic order.)